monkeys/src/main/kotlin/com/wire/kalium/monkeys/homeDirectory.kt

```diff
@@ -34,7 +34,6 @@ fun coreLogic(
             encryptProteusStorage = true,
             isMLSSupportEnabled = true,
             wipeOnDeviceRemoval = true,
-            fetchAllTeamMembersEagerly = true,
```

Comment:
btw there is a new value 
`limitTeamMembersFetchDuringSlowSync` if you set it to null is will behave the same as if `          fetchAllTeamMembersEagerly = true,`